modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -282,8 +264,12 @@ export default {
         this.editing = this.items.find(item => item._id === id);
         this.editsInProgress = false;
```

Comment:
This and the declaration on L 262 should also use `updateEditsInProgress`

Reply:
Actually the purpose is to distinguish between updates coming from AposSchemas (or otherwise from the user) and those not (which would trigger false updates of `modified`). That's why I needed the event handler and couldn't just `watch`.

Reply:
`editsInProgress` is semantically the same as `modified` and served that purpose in the initial implementation of the confirmation modal. `editsInProgress` should probably be retired with this PR.